runtime/rasdump/dmpmap.c

```diff
@@ -70,7 +70,14 @@ static const DgDumpAction dgActions[] = {
 
 static const int numDumpActions = ( sizeof(dgActions) / sizeof(DgDumpAction) );
 
+/* these are the filter strings used for the default OutOfMemory dump options */
+#define  SYSTHROW_OOM_1_1 "events=systhrow,range=1..1,filter=java/lang/OutOfMemoryError,request=exclusive+compact+prepwalk"
+#define  SYSTHROW_OOM_1_4 "events=systhrow,range=1..4,filter=java/lang/OutOfMemoryError"
 
+#define HEAP_KIND "heap"
+#define JAVA_KIND "java"
+#define SNAP_KIND "snap"
+#define SYSTEM_KIND "system"
```

Comment:
Nit: Can we have a blank line after these definitions? The aren't directly connected to the following type.

Reply:
Done.